package.json

```diff
@@ -29,9 +29,9 @@
     "url": "https://github.com/w3c/ghapi/issues"
   },
   "dependencies": {
+    "cli-progress-bar": "1.0.1",
     "express": "4.14.0",
-    "octonode": "0.7.6",
-    "cli-progress-bar": "1.0.0"
+    "octonode": "tripu/octonode"
```

Comment:
Well spotted, @deniak. I think I just found a bug in that PR of mine though; I'm working on a fix for that now.